FancyMouse/Network/Data/WordResponse.swift

```diff
@@ -6,3 +6,36 @@
 //
 
 import Foundation
+
+struct WordResponse: Decodable {
+    let wordID: Int
+    let folderID: Int
+    let createdAt: Date
+    let spelling: String
+    let meanings: [String]
+    let status: Int
```

Comment:
`Word.Status` 를 사용하기 위해서 Response에서 Int로 받아서 `init(_ number: Int)`로 변환하시는 것 같은데요
 `Word.Status`를 `init(from decoder: Decoder) throws`구현해서 decodable로 만들면 `WordResponse`에서는 `Word.Status`타입으로 표기할 수 있어서 의도가 조금 더 명확해 질 것 같아요
그러면 `WordResponse.mappedWord` extension도 지울 수 있지 않을까용

https://www.untitledkingdom.com/blog/codable-enums-in-swift-3ab3dacf30ce 참고하시면 좋을 것 같습니당
 ```suggestion
     let status: Word.Status
```

Reply:
리뷰 감사합니다! 🙇‍♀️👍
DTO랑 Entity와 구분하기 위해서 mappedWord를 쓴건데 혹시 그것과 별개로 WordStatusResponse라는 decodable enum을 만들 필요가 있다는 말씀일까요?🥺

Reply:
아하 WordResponse보다 Word.Status가 조금 더 바깥쪽 계층의 객체라고 이해하면 될까요? 제가 같은 계층이라고 생각하고 커멘트를 남겨서 헷갈렸던 것 같습니다. 😥
그렇지만 제가 말하고 싶었던 내용은 서버에서 받은 status를 조금 더 명확하게 0은 ready의 의미를 가지고 있고 1은 inProgress의 의미를 가지고 있다고 Decodable에 명시할 수 있으면 읽는 사람이 이해하기 쉽지 않을까 였던 거였어요
WordStatusResponse를 따로 만들고 Int를 enum으로 바꾸는 로직을 Word.Status에서 가져오는건 어떨까요?